src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num, res) {
```

Comment:
it's not clear why one of the operand is `num`, and second one is `res`, will be better to name them in more general way without any relation to some number or result property